challenges/08-coding-interview-prep/rosetta-code.json

```diff
@@ -12434,6 +12434,1022 @@
+      ],
+      "tests": [
+        {
+          "text": "'<code>pascal</code> should be a function.'",
+          "testString":
+            "assert(typeof pascal=='function','<code>pascal</code> should be a function.');"
+        },
+        {
+          "text": "'<code>pascal(11,4,40,151)</code> should return a array.'",
+          "testString":
+            "assert(Array.isArray(pascal(11,4,40,151)),'<code>pascal(11,4,40,151)</code> should return a array.');"
+        },
+        {
+          "text":
+            "'<code>pascal(11,4,40,151)</code> should return <code>'+JSON.stringify(results[0])+'</code>.'",
+          "testString":
+            "assert.deepEqual(pascal(11,4,40,151),results[0],'<code>pascal(11,4,40,151)</code> should return <code>'+JSON.stringify(results[0])+'</code>.');"
+        },
+        {
+          "text":
+            "'<code>pascal(11,4,41,151)</code> should return <code>'+JSON.stringify(results[1])+'</code>.'",
+          "testString":
+            "assert.deepEqual(pascal(11,4,41,151),results[1],'<code>pascal(11,4,41,151)</code> should return <code>'+JSON.stringify(results[1])+'</code>.');"
+        },
+        {
+          "text":
+            "'<code>pascal(11,4,35,151)</code> should return <code>'+JSON.stringify(results[2])+'</code>.'",
+          "testString":
+            "assert.deepEqual(pascal(11,4,35,151),results[2],'<code>pascal(11,4,35,151)</code> should return <code>'+JSON.stringify(results[2])+'</code>.');"
+        },
+        {
+          "text":
+            "'<code>pascal(11,4,440,151)</code> should return <code>'+JSON.stringify(results[3])+'</code>.'",
+          "testString":
+            "assert.deepEqual(pascal(11,4,440,151),results[3],'<code>pascal(11,4,440,151)</code> should return <code>'+JSON.stringify(results[3])+'</code>.');"
+        }
+      ],
+      "id": "5a23c84252665b21eecc7f5e",
+      "challengeType": 5,
+      "releasedOn": "August 15, 2018",
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": [
+            "function pascal (a, b, mid, top) {",
+            "  // Good luck!",
+            "}"
+          ],
+          "head": [],
+          "tail": [
+            "var results=[",
+            "[5,13,8],",
+            "[6,13,7],",
+            "[0,13,13],",
+            "[405,13,-392]]"
+          ]
+        }
+      }
+    },
+    {
+      "title": "Pascal's triangle",
+      "description": [
+        "<a href=\"https://en.wikipedia.org/wiki/Pascal's triangle\">Pascal's triangle</a>  is an arithmetic and geometric figure first imagined by  <a href=\"https://en.wikipedia.org/wiki/Blaise Pascal\">Blaise Pascal</a>.",
+        "Its first few rows look like this:",
+        "1",
```

Comment:
Could this be formatted using `<pre>` tags like on the Rosetta Code site? So something like:

`"<pre>   1\n  1 1\n 1 2 1\n1 3 3 1</pre>"`